src/prof/src/http/mod.rs

```diff
@@ -84,7 +84,12 @@ struct FlamegraphTemplate<'a> {
 }
 
 #[allow(clippy::drop_copy)]
-async fn time_prof<'a>(merge_threads: bool, build_info: &BuildInfo) -> impl IntoResponse {
+async fn time_prof<'a>(
+    merge_threads: bool,
+    build_info: &BuildInfo,
+    time_secs: u64,
+    sample_freq: u32,
```

Comment:
`sample_freq_us`?

Reply:
That's not what it is though, the unit is Hz, not µs